core/dnsserver/zdirectives.go

```diff
@@ -50,5 +50,6 @@ var Directives = []string{
 	"grpc",
 	"erratic",
 	"whoami",
+	"acl",
```

Comment:
`acl` plugin should be higher in this list. I would say it should be immediately after `dnstap`

Reply:
Just curious: shouldn't `acl` works before `forward`?

Reply:
Yes, it should. The higher plugin in this list, the earlier it takes control.

Reply:
https://github.com/coredns/coredns/blob/master/plugin.cfg#L4

> # Ordering is VERY important. Every plugin will
> # feel the effects of all other plugin below
> # (after) them during a request, but they must not
> # care what plugin above them are doing.

When I saw this, I thought CoreDNS chains the plugins in reverse order...
Will change this.